package.json

```diff
@@ -56,7 +56,9 @@
     "@fortawesome/free-brands-svg-icons": "6.4.2",
     "@fortawesome/free-regular-svg-icons": "6.4.2",
     "@fortawesome/free-solid-svg-icons": "6.4.2",
-    "@fortawesome/react-fontawesome": "0.2.0"
+    "@fortawesome/react-fontawesome": "0.2.0",
+    "@openedx-plugins/footer-links": "file:../src/plugins/footer-links",
```

Comment:
from the changes you made already, should this still be here?

Reply:
Yes, the plugin is designed to display footer links by default upon installation. Since it is not separately available on `npm`, that's why included it like that .If there will be a requirement to hide these links, users have the flexibility to do so using the `env.config.js` file. Or Is there any better way to handle this, we can go ahead with that?

Reply:
With `frontend-plugin-framework`, the plugins don't _have_ to be installed as separate npm packages though. Since the source files are already part of this repository, you can just specify the plugin path using the config file. At least, that's how I thought it worked.

Reply:
That's right @bradenmacdonald ! I've been importing a component purely from the local path needed to fetch it from the repo, but I've also seen a configuration in Learning MFE to do some [aliasing of the plugins](https://github.com/openedx/frontend-app-learning/pull/1330/files#diff-740143daa3776af3043426a70a9d72908830d126689ba30aed659280ba2a361d), which I imagine would help in the case where we want to clearly mark a component's purpose as being a "plugin". 

With that, I don't think this `package.json` approach is necessary.

Reply:
Makes sense. I also got the idea of aliasing of plugins from [`course-authoring` MFE](https://github.com/openedx/frontend-app-course-authoring/pull/638). I've removed it. Also, I want to ask are we going to include links in footer? I've read that [ADR](https://github.com/openedx/frontend-component-footer/pull/326) and that's why included the footer-links.